multi-cluster-orchestrator/samples/hello_world/README.md

```diff
@@ -0,0 +1,420 @@
+
+## Before you begin
+
+Take the following steps to enable the Kubernetes Engine API:
+
+  1. In the Google Cloud console, on the project selector page, select or [create a
+  Google Cloud project][2].
+
+  > [!TIP]
+  > If you don't plan to keep the resources that you create in this procedure,
+  > create a project instead of selecting an existing project. After you finish
+  > these steps, you can delete the project, removing all resources associated with
+  > the project.
+
+1. [Make sure that billing is enabled for your Google Cloud
+  project][3].
+
+1. [Enable the following services][4]:
+   1. Compute Engine
+   1. Google Kubernetes Engine
+   1. GKE Hub
+   1. Connect Gateway
+   1. Monitoring
+   1. Traffic Director
+   1. Multi-cluster Ingress
+   1. Multi-cluster Service Discovery
+
+1. In addition to the `Owner` role, your account will need the Service Account
+  Token Creator role (`roles/iam.serviceAccountTokenCreator`).
+
+You should be familiar with the basics of Terraform. You can use the following
+resources:
+
+- [Getting Started with Terraform][5] (video)
+- [Terraform commands][6]
+
+## Prepare the environment
+
+In this tutorial, you use [Cloud Shell][7] to manage resources
+hosted on Google Cloud. Cloud Shell is preinstalled with the software you need
+for this tutorial, including Docker, [Terraform][8],
+[kubectl][9], and the [gcloud CLI][10].
+
+> [!WARNING]
+> If you do not use Cloud Shell, you may need to install Docker,
+> Terraform, kubectl and the gcloud CLI. You must also set your default
+> Terraform project with: `export GOOGLE_CLOUD_PROJECT=PROJECT_ID`.
+
+1. Launch a Cloud Shell session from the Google Cloud console, by clicking Cloud
+Shell activation icon Activate Cloud Shell in the Google Cloud console. This
+launches a session in the bottom pane of the Google Cloud console.
+
+The service credentials associated with this virtual machine are automatic, so
+you do not have to set up or download a service account key.
+
+2. Before you run commands, set your default [project][11] in the Google Cloud CLI
+  using the following command:
+
+```
+gcloud config set project PROJECT_ID
+```
+
+3. Clone the GitHub repository:
+
+```
+git clone https://github.com/GoogleCloudPlatform/gke-fleet-management.git --single-branch
+```
+
+4. Change to the Hello World sample directory:
+
+```
+cd gke-fleet-management/multi-cluster-orchestrator/samples/hello_world
+```
+
+## Review the Terraform file
+
+The [Google Cloud Platform Provider][12] is a plugin that lets you
+manage and provision Google Cloud resources using Terraform, HashiCorp's
+Infrastructure as Code (IaC) tool. The Google Cloud Platform Provider serves as a
+bridge between Terraform configurations and the Google Cloud APIs, letting you
+define infrastructure resources, such as virtual machines and networks, in a
+declarative manner.
+
+1. Review the following Terraform file:
+
+```
+cat 1-infrastructure/main.tf
+```
+
+This file describes the following resources:
```

Comment:
should we mention that this is the step that the platform would own.

Reply:
Sure, added